frontend/src/components/common/LanguageSwitcher.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import { locale, locales } from 'svelte-i18n';
+  import { isLoading } from 'svelte-i18n';
+
+  function handleChange(event: Event) {
+    const selectedLocale = (event.target as HTMLSelectElement).value;
+    localStorage.setItem('locale', selectedLocale);
+  }
+
+  if ($locale != null && $locale != undefined && $locale.startsWith('ru')) {
```

Comment:
Let's use `$locale && $locale.starts...` instead of checking for `null` and `undefined`